src/es-module-shims.js

```diff
@@ -267,15 +267,18 @@ function resolveDeps (load, seen) {
         if (cycleShell) {
           // circular shell creation
           if (!(blobUrl = depLoad.s)) {
+            function dotOrBracketAccessor(expt) {
+              const ch = depLoad.S[expt.s];
+              return (ch === '"' || ch === "'") ? `[${depLoad.S.slice(expt.s, expt.e)}]` : '.' + expt.n;
+            }
+            const shellExports = depLoad.a[1].map(expt => expt.ln ? expt : {...expt, ln: 'd$_'});
```

Comment:
For early default updates, we should instead explicitly detect `expt.n === 'default'`.

Reply:
I still don't see how we would update the default without a local name though... I'm pretty sure it's a moot point.

Reply:
I went ahead and added the ordered binding approach discussed previously!